sql/util/wasm_initial_set.sql

```diff
@@ -0,0 +1,7 @@
+SELECT DISTINCT
+  url
```

Comment:
```suggestion
  _TABLE_SUFFIX AS client,
  url
```

Reply:
This one is def not supposed to be grouped because it's an input list for downloader which is meant to contain only distinct URLs.